ludwig/schema/defaults/defaults.py

```diff
@@ -37,9 +37,4 @@ class DefaultsConfig(schema_utils.BaseMarshmallowConfig):
 def get_defaults_jsonschema():
     """Returns a JSON schema structured to only require a `type` key and then conditionally apply a corresponding
     combiner's field constraints."""
-    defaults_schema = schema_utils.unload_jsonschema_from_marshmallow_class(DefaultsConfig)
```

Comment:
@connor-mccorm sanity check: did this add anything?

Reply:
Lol no, that's some dumb code right there!